ingest/lastfm/schema.py

```
@@ -36,16 +36,24 @@ def initialize_mart(conn: duckdb.DuckDBPyConnection, tracks_glob: str, artists_g
         # スキーマの存在を確認
         conn.execute("CREATE SCHEMA IF NOT EXISTS mart")
 
-        # トラックビューの作成
+        # トラックビューの作成（プレースホルダーではなく直接埋め込み）
         try:
-            conn.execute(LastFmSchema.MART_TRACKS_VIEW, [tracks_glob])
+            tracks_view_sql = f"""
+                CREATE OR REPLACE VIEW mart.lastfm_tracks AS
+                SELECT * FROM read_parquet('{tracks_glob}', hive_partitioning = 1)
+            """
```

Ruff (0.14.10)

Possible SQL injection vector through string-based query construction

(S608)